static/js/timerender.js

```diff
@@ -164,6 +164,41 @@ exports.render_date = function (time, time_above, today) {
     return node;
 };
 
+// Renders the timestamp returned by the !time() markdown syntax.
+exports.render_timestamp = function (time, now, text) {
+    now = now || moment();
+    if (page_params.timezone) {
+        now = now.tz(page_params.timezone);
+        time = time.tz(page_params.timezone);
+    }
```

Comment:
We don't want to use `page_params.timezone` here; I'd prefer to use the browser's local time as we do for all the other `timerender` methods.  `page_params.timezone` is oddly only used for display to other users.